tests/runtest.cmd

```diff
@@ -333,25 +337,62 @@ goto :eof
     powershell -Command Write-Host %1 -foreground "red"
     exit /b -1
 
+:RestoreCoreFXTests
+
+    :: Explicitly restore the test helper project
+    "%CoreRT_CliDir%\dotnet.exe" msbuild /t:Restore "%CoreRT_TestFileHelperProjectPath%"
```

Comment:
Should we check error level after this too?

Reply:
Added.